### ISSUE TITLE
DisplayName not retreived by default from latest React APIs

### ISSUE THREAD

**Issue by pascalduez** [algolia/react-element-to-jsx-string#352]
```js
function Foo() {
  return <div />
}

export default React.memo(Foo);
```

Results in:
```
<[object Object] />
```

See https://github.com/facebook/react/issues/14319

Opening an issue here to track the progress, and see what we can do.  
The thing is we need the displayName not only for DEV env, but also production.

**Comment by Spy-Seth** [comment]
We are always happy to receive PR 😄

**Comment by mallow-fight** [comment]
when to fix this bug? does any method to solve it?

**Comment by armandabric** [comment]
This issue seems to have been fixed by #617. If not the case you could re-open the issue.
